src/read.c

```diff
@@ -2572,6 +2572,18 @@ void ReadReturn (
     }
 }
 
+void ReadPragma(TypSymbolSet follow)
+{
+    Obj  pragma;
+    UInt len;
+
+    C_NEW_STRING( pragma, STATE(ValueLen), (void *)STATE(Value) );
+    len = STATE(ValueLen);
+    SET_LEN_STRING(pragma, len);
+    *(CHARS_STRING(pragma) + len) = 0;
```

Comment:
`C_NEW_STRING ` and `NEW_STRING ` actually already call `SET_LEN_STRING`, and add a zero terminator, so the previous three lines are not needed.